pkg/util/log/eventpb/telemetry.proto

```diff
@@ -109,3 +110,16 @@ message ChangefeedFailed {
   // connection_closed, changefeed_behind)
   string failure_type = 2 [(gogoproto.jsontag) = ",omitempty", (gogoproto.moretags) =  "redact:\"nonsensitive\""];
 }
+
+// Schema is an event for schema telemetry.
+message Schema {
+  CommonEventDetails common = 1 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "", (gogoproto.embed) = true];
+  uint32 current_page = 2;
```

Comment:
Now that we're using the descriptors, what are these pages providing? I do see that it might let you establish a snapshot.  Can we have some commentary about them? Should we add some unique key that we might be able to use to tie together the events which go together? I recognize that we could do something about trying to leverage the events with timestamp near each other. I don't know if that'll be convenient or painful.

Reply:
> Can we have some commentary about them?

Sure, will do.

> Should we add some unique key that we might be able to use to tie together the events which go together?

I believe the `timestamp` in `common` already fulfills that purpose, so I didn't add anything else.